cea/analysis/sensitivity/sensitivity_demand_analyze.py

```diff
@@ -0,0 +1,94 @@
+"""
+Analyze the results in the samples folder and write them out to an Excel file. This script assumes:
+
+- all the results have been added to `--samples-folder` in the format `result.%i.csv`, with `%i` replaced by the index
+  into the samples array.
+- each result file has the same list of columns (the `--output-parameters` for the simulations were the same)
+- the `analyze_sensitivity` function is called with the same method and arguments as the sampling routine.
+"""
+import os
+import numpy as np
+import pickle
+
```

Comment:
all this script needs comments in line and documentation of input and output variables